### DISCUSSION TITLE
p-menubar items go behind the other content

### DISCUSSION THREAD

**Discussion by Sampath-Lokuge** [primefaces/community#1241] [PrimeNG] [answered]
Hello,

![pri](https://github.com/primefaces/community/assets/5206526/db78fce0-60ab-436a-92d3-24921b59c230)

How can I show the menu items without going behind the other content as shown above?

```
<div class="margin-top-25">
          <p-menubar [model]="items"></p-menubar>
        </div>
```

  **_"primeng": "16.2.0",_**

**Comment by cetincakiroglu** [comment]
Hi,

You may solve this by providing baseZIndex value to the menubar. To understand and identify the issue better, could you please provide a stackblitz example?

**Comment by Sampath-Lokuge** [comment, marked as answer]
Hi @cetincakiroglu 

I have solved it by increasing the height of the inner div. It works for me. Thanks for your feedback too.
